### PR TITLE
Migrate php to 20.04

### PR DESCRIPTION
**Pull request by dsame**
# Description
New tool, Bug fixing, or Improvement?  
Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
**For new tools, please provide total size and installation time.**

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/2004/php.sh [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/etc-environment.sh
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/os.sh
+
+set -e
+
+# Install PHP
+
+if isUbuntu20 ; then
```
Comment (AlenaSviridenko) [inline images/linux/scripts/installers/2004/php.sh:16]: I think this condition is not needed because this file in Ubuntu 20 folder

Comment (dsame) [reply images/linux/scripts/installers/2004/php.sh]: Agree

File: images/linux/scripts/installers/php.sh [hunk 2]
```diff
@@ -10,6 +10,12 @@ source $HELPER_SCRIPTS/document.sh
 source $HELPER_SCRIPTS/os.sh
 
 set -e
+# since 20.04 php is installed from Ubuntu repository
+# see https://github.com/actions/virtual-environments/issues/1084
+if isUbuntu20 ; then
```
Comment (al-cheb) [inline images/linux/scripts/installers/php.sh:15]: Why do you split `php.sh` if you can simple add `if` statement?

```
if isUbuntu16 || isUbuntu18; then
    # add repository
    apt-add-repository ppa:ondrej/php -y
    apt-get update
fi
```

**Comment by shivammathur** [comment]
This approach is even worse than #1104, as installing PHP 7.4 without ondrej's PPA on focal would mean different patch releases for PHP 7.4 between bionic and focal. Please see https://github.com/actions/virtual-environments/pull/1104#issuecomment-650711664.

**Comment by dsame** [comment]
>  between bionic and focal

but why don't they stay with bionic if they depends on ondrej's PPA so much?

File: images/linux/scripts/installers/php.sh (hunk 2 above)
Comment (dsame) [reply images/linux/scripts/installers/php.sh:15]: Using the other repo seems to be a very significant change which should not be handled by "if" branches. Different package names and such. I just created the VM and testing

**Pull request closed without merging**